docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1429,6 +1429,12 @@ The following table describes the JSON fields of the CADF audit log entries logg
 |`observer/geolocation/city`                           |Value is determined by "fhirServer/audit/serviceProperties/geoCity" configuration property.|
 |`observer/geolocation/state`                          |Value is determined by "fhirServer/audit/serviceProperties/geoState" configuration property.|
 |`observer/geolocation/region`                         |Value is determined by "fhirServer/audit/serviceProperties/geoCounty" configuration property.|
+|`target/id`                                           |Value is the UUID from the resource|
```

Comment:
is this an internal UUID or is this the logical id?  if its the logical id, then its not necessarily a UUID.

Reply:
Fixed